paddle_billing/Entities/CreditNote.py

```diff
@@ -0,0 +1,15 @@
+from __future__  import annotations
+from dataclasses import dataclass
+
+from paddle_billing.Entities.Entity import Entity
+
+
+@dataclass
+class CreditNote(Entity):
```

Comment:
I was going with `AdjustmentCreditNote` though I'm not against just having it as `CreditNote` ... what do you think?